auth_test.go

```diff
@@ -925,3 +925,21 @@ func TestOktaRetryWithNewToken(t *testing.T) {
 	assertEqualF(t, authResponse.MfaToken, expectedMfaToken)
 	assertEqualF(t, authResponse.SessionInfo.DatabaseName, expectedDatabaseName)
 }
+
+func TestGetAccountForJWT(t *testing.T) {
+	accounts := map[string]string{
```

Comment:
`testcases` is a name we usually use in this driver.

Reply:
renamed